packages/ra-ui-materialui/package.json

```diff
@@ -37,6 +37,7 @@
         "@types/react-is": "^18.3.0",
         "@types/react-transition-group": "^4.4.10",
         "cross-env": "^5.2.0",
+        "csstype": "^3.1.3",
```

Comment:
This dep is actually used in the code and was missing from the `package.json` file.

https://github.com/marmelab/react-admin/blob/a4703fd2e29c3ad849cde0d6d31bd5767027385f/packages/ra-ui-materialui/src/Labeled.tsx#L11